cliche/services/tvtropes/crawler.py

```diff
@@ -133,23 +133,20 @@ def fetch_link(url, session, *, log_prefix=''):
         return False, None, None, None, final_url
     tree = document_fromstring(r.text)
     try:
-        namespace = tree.xpath('//div[@class="pagetitle"]')[0] \
-            .text.strip()[:-1]
+        name = (tree.find_class('article_title')[0]).text_content()
     except (AttributeError, AssertionError, IndexError):
-        logger.warning('%sWarning on url %s: '
-                       'There is no pagetitle on this page. Ignoring.',
-                       log_prefix, url)
+        logger.warning('no name in %s', final_url)
```

Comment:
It's just a matter of taste, but I feel that this error message lacks a bit of information. + capitalization